modules/pipeline/pipengine/reconciler/task.go

```diff
@@ -83,24 +84,32 @@ func reconcileTask(tr *taskrun.TaskRun) error {
 			// Do 没有 err 才是正常的，即使失败也是 status=Failed，没有 error
 			abnormalErr := tr.Do(taskOp)
 			if abnormalErr != nil {
-				rlog.TWarnf(tr.P.ID, tr.Task.ID, "failed to handle taskOp: %s, abnormalErr: %v, continue retry", taskOp.Op(), abnormalErr)
+				if errors.IsContainUserError(abnormalErr) {
```

Comment:
Can you use IsUserError directly?

Reply:
![image](https://user-images.githubusercontent.com/30427474/129848055-1f4e0035-0869-4f0d-a0c0-0e342671b4d1.png)
task.waitOp aggregate the errors list to one error, the aggregate error may contain platform error and user error